Makefile.in

```diff
@@ -5,11 +5,11 @@ dep = $(obj:.o=.d)
 bin = spacenavd
 ctl = spnavd_ctl
 
-CC = gcc
-INSTALL = install
-CFLAGS = -pedantic -Wall $(dbg) $(opt) -fno-strict-aliasing -fcommon \
+CC ?= gcc
+INSTALL ?= install
+CFLAGS += -pedantic -Wall $(dbg) $(opt) -fno-strict-aliasing -fcommon \
 		 -I$(srcdir)/src -I/usr/local/include -MMD $(add_cflags)
-LDFLAGS = -L/usr/local/lib $(xlib) $(add_ldflags)
+LDFLAGS += -L/usr/local/lib $(xlib) $(add_ldflags)
 
```

Comment:
I think these are best handled during configure. In fact CFLAGS and LDFLAGS are already passed on to the makefile if set during the execution of the configure script. Maybe that mechanism should be expanded for CC and INSTALL as well?

To be honest, using install is pretty pointless, I should probably just change that to use cp instead.

More importantly, these have nothing to do with FreeBSD, and should not be bundled into this pull request.

Reply:
CC is a standard predefined variable, and gcc isn't installed on FreeBSD by default.  Simply removing CC would likely be the best solution, but I assumed it was there for a reason.

As for the INSTALL/LDFLAGS/CFLAGS, those changes were mostly to make writing a FreeBSD port easier.  The main reasons to use install is to be able to strip the binary, set the ownership/permissions, and simple DESTDIR support.